README.md

```diff
@@ -7,7 +7,7 @@ This module creates a private ALB, a default ALB listener (with HTTP protocol),
 ```tf
 module "alb" {
   source  = "babbel/alb-for-vpc-internal-requests/aws"
-  version = "~> 1.1"
+  version = "~> 1.1.1"
```

Comment:
Will will prevent minor version updates, e.g. to `1.2`. Why is this change necessary to use `1.1.1`?